title_case/title_case_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rspec'
+require_relative 'title_case'
+
+describe 'title_case' do
+  it 'should capitalize words' do
```

Comment:
`nitpick` we could omit `should`, and use the third person. [betterspecs](http://www.betterspecs.org/#should).